src/webui/api/torrentscontroller.cpp

```diff
@@ -150,6 +150,7 @@ namespace
 // GET params:
 //   - filter (string): all, downloading, seeding, completed, paused, resumed, active, inactive
 //   - category (string): torrent category for filtering by it (empty string means "uncategorized"; no "category" param presented means "any category")
+//   - hashes (string): filter by hashes, can contain multiple hashes separated by |
```

Comment:
You'll be required to update both wiki pages after this PR is accepted & merged:
https://github.com/qbittorrent/qBittorrent/wiki/Web-API-Documentation
https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-Documentation

Reply:
Will do!